src/test/scala/scala/tools/refactoring/tests/util/TextSelections.scala

```diff
@@ -0,0 +1,127 @@
+package scala.tools.refactoring.tests.util
+
+import scala.collection.immutable.Queue
+import scala.annotation.tailrec
+
+/**
+ * Helper class to extract selections from unit test input.
+ *
+ * The following markers are supported:
+ *
+ * <ul>
```

Comment:
Hmm, you mean something like
```scala
 /**
   * Title
   *
   * * Item 1
   * * Item 2
   */
```

At least in the ScalaIDE hovers, this does not work for me....